### ISSUE TITLE
Contract

### ISSUE THREAD

**Issue by ryanatk** [ryanatk/junction#2]
I've put together a contract using a popular template for web desing & development work
https://github.com/ryanatk/junction/blob/master/docs/contract.md

It's basic, using plain language, and covers all the bases. I've updated it with our specifics. Please let me know if you have any questions or changes you'd like to see.

I can upload this for us to e-sign, or you can print, sign, and scan. Whichever is easier for you.

I hope we can both sign this week, and then get started on the main site.

**Comment by mishmosh525** [comment]
Ryan,

Thanks for putting all this together and also meeting with Will last week. Sorry we weren't able to meet! Generally speaking this all looks good. A couple of items that I'd like to address:

- **Contact Forms** - We will need contact forms for various inquiries, similar to the local's page (http://thelocal-cville.com/ hover over "contact us to see the different forms). These will need to generate into an email that is sent to us. I'll forward you a sample.
- **Email** - Not sure if this is your realm, but we need to have email addresses set up. Let me know if you have any information there. Right now we use squirrel mail for The Local with everything forwarded to gmail.
- **The Local Website** - Can we incorporate the template building of The Local's page into this contract? Ideally this wouldn't be a new build, but just recreating the Junction page with new logos, photos, text etc. Hoping this would be included or if not included minimal cost to set up.

Thanks!

**Comment by ryanatk** [comment]
Hi Michelle

Thanks for the further explanation. Will & I did not include the 3 items you listed above in the phase of this project, so I'll address them each here:

## Contact Forms

We discussed initially forgoing a contact page, in favor of adding a [contact me](mailto:ryana804@gmail.com) link to the footer. That link opens a new email with the "To:" address already filled in.

Our logic was that these forms may not immediately be essential, and could be added later (more explanation below). However, if you feel want these now, we can certainly add them by adjusting the contract.

## Email

I'm sorry. I frankly don't have much experience with managing email addresses with custom domains or SquirrelMail, so I'm likely not the best option. However, if you don't have another option, we can also add this to the contract.

## The Local Website

I would love to also build a new website for The Local. You're correct that we can build The Local site the same way we build Junction, saving us from completely starting from scratch, but it's not quite as simple as copy+pasting the Junction website and swapping out images. There are some good reasons to complete the Junction site first:

1. If we develop in parallel, any changes we make will need to be made twice. If we finish Junction first, then that lays a plan for The Local, and helps us avoid repeating those same mistakes & fixes.
2. Developing them both at the same time will delay the Junction site's launch. The Local already has a site, so it could reasonably stay with that a bit longer. It's also likely that developing both at the same time would take longer than one after the other (see #1 above).
3. It appears that The Local website includes more content and functionality than is necessary for the Junction site, at least until the Junction restaurant is fully up & running. This includes: Bottle List, Catering Menu, Gift Certificates, Singer/Songwriter Events, etc. The more time we spend up front reconciling what is and isn't necessary for The Local site, the more we delay the Junction launch.
4. A smaller initial commitment can cover us both, moving forward. After completing this first contract, we can adjust future agreements & expectations accordingly.

I suggest that we split this into 3 pieces, done 1-at-a-time:

1. Build the initial Junction site with everything useful now
2. Recreate The Local site, using the same layout & tools as Junction, incorporating all content & functionality from current site
3. At this point Junction will have press releases, reviews, more photos for the gallery, events, catering info, etc, so we can add this content & functionality to the Junction site (catching it up with The Local)

I understand that a goal was to avoid ongoing development, but I believe that breaking this project into smaller these pieces gives each piece the best chance for success. Most importantly, it gives the Junction website it's best chance for success.

Please let me know if you have any questions or feedback

Thank you
Ryan

**Comment by mishmosh525** [comment]
Ryan,

We will definitely want the contact forms at the start, as the main driver
of the webpage is to bring in inquiries for large parties. We can start
with just three if that is easier (private events, catering, general), but
that is essential purpose of the website from a revenue standpoint. We will
need the custom fields to match The Local as opposed to a generic email
pop-up, as the pop-up will just require us to respond back requesting the
required information anyway.

No worries on the email, wasn't sure if they were related but I can get
that sorted out!

To clarify, definitely not interested in having them be built at the same
time, but more what I'm looking for is a comprehensive quote including
both. I will have a hard time rationalizing $2k just for the Junction
website, but if for example the work to replicate the site is $500
additional for The Local, I would have an easier time selling $2,500 for
both. Let me know if that makes sense? I would still want to focus all the
attention to Junction first of course, and then to The Local's site.

Michelle Moshier
General Manager

*The Local Restaurant & Catering*
824 Hinton Ave, Charlottesville, VA 22902
434.218.3660 (direct)
434.984.9749 (restaurant)
michelle@thelocal-cville.com

**Comment by ryanatk** [comment]
Hi Michelle

Gotcha. We can include all the contact forms from the start. It makes sense now that those are a priority. Once we get started, we can discuss details: should they all be on one page, what we'll want to call the page(s) in the top navigation, etc.

As for The Local, I don't think that price is reasonable. This is based on my assumption that a new site should include all information currently on [the existing website](http://thelocal-cville.com/), which is significantly more than [what we've laid out for the Junction site](https://github.com/ryanatk/junction/wiki/Site-Structure). So while I could use much of the same bones from Junction, I'll still need to implement them separately and then build on top. If that additional info is no longer necessary, then we can definitely discuss a discounted rate. Otherwise I anticipate the price to be around the same as the Junction website.

I feel more comfortable limiting this initial contract to just the Junction site, with good faith that we will discuss The Local site when it comes time. If we find thru this project that we've estimated the work inaccurately, then we can absolutely adjust any future contracts. This also provides you the opportunity to build confidence in what I provide before further committing.

I'm sorry if this is bad news. It wasn't an easy email to write. I'm sincerely eager to work with you, and I hope we can start soon.

Thank you
Ryan

**Comment by mishmosh525** [comment]
Hey Ryan,

Maybe we can have a quick chat so I can clarify what we are looking for in
The Local site? We don't actually want all the info that is on The Local
site. In fact I'd love to completely get ride of that and have it set up
just like we have Junction. The only area that would be a bit more robust
is some "about us" type pages. So as far as the work involved, we would
create content and navigation that would mimic Junction's layout, which to
me seems like it would be a fairly straightforward development. Change
logos, fonts, and maybe color. Add maybe 1-2 tabs, but that would be it.

Hopefully that makes sense, but happy to talk through it over the phone. I
would like to have pricing locked in for both, even if The Local site
estimate is a range and will be honed in after the Junction site is done.

Thanks!

Michelle Moshier
General Manager

*The Local Restaurant & Catering*
824 Hinton Ave, Charlottesville, VA 22902
434.218.3660 (direct)
434.984.9749 (restaurant)
michelle@thelocal-cville.com

**Comment by ryanatk** [comment]
Ah, ok, that makes more sense. That's my fault for looking ahead and assuming we'd need to implement everything from The Local. You're correct, as long as we keep the template and the navigation close to the same, that should save a considerable amount of time to setup. Thank you for clearing that up!

Using the same template will save time on up front "planning & design". This should also help us avoid many bug fixes or "gotchas", after seeing those the first time around. We won't save much time with the actual development part of the work, because I'll still need to manually set up everything. And you will still have opportunity to review and give feedback, the same as with Junction.

All told, I can agree to do The Local site for half the price of the Junction site, for a total of $3,000 for both sites. Are you comfortable with me updating the contract with this price? Our deadline for the Junction site is end of December. Should we plan to complete The Local by end of January? And are there any other edits you want to see on [the contract itself](https://github.com/ryanatk/junction/blob/master/docs/contract.md)?

I'd love to talk on the phone so we can do proper introductions, but I woke up this morning feeling under the weather. Is one day later this week ok with you?

Thanks again
Ryan

**Comment by mishmosh525** [comment]
​Ryan,

That all sounds​ great. Contract looks good with that change. Also the
addresses for Hawthorne Hospitality and The Local are 824 Hinton Avenue,
and Junction is 421 Monticello Rd. Not sure if it matters, but might as
well have the right info!

Thanks!

Michelle Moshier
General Manager

*The Local Restaurant & Catering*
824 Hinton Ave, Charlottesville, VA 22902
434.218.3660 (direct)
434.984.9749 (restaurant)
michelle@thelocal-cville.com

**Comment by ryanatk** [comment]
Awesome! I've updated the contract
https://github.com/ryanatk/junction/blob/master/docs/contract.md

And I uploaded to "hellosign", if you'd like to e-sign. I'm also attaching a signed copy here, if you'd rather print & sign.
[Junction___Local_website_contract.pdf](https://github.com/ryanatk/junction/files/635288/Junction___Local_website_contract.pdf)

I've also sent the first invoice using **freshbooks**. Please let me know if you have any questions regarding the invoice or freshbooks. I haven't had any issues with it, but I have not been using it for long.

I have enough information to start work on the Junction site this week. I'll begin by building out the [Site Structure](https://github.com/ryanatk/junction/wiki/Site-Structure), so I can get feedback from you next week.

Thank you
Ryan

**Comment by ryanatk** [comment]
[Junction___Local_website_contract (1).pdf](https://github.com/ryanatk/junction/files/644169/Junction___Local_website_contract.1.pdf)
